apis/proto/v1/payload/payload.proto

```diff
@@ -89,6 +89,8 @@ message Search {
     Filter.Config ingress_filters = 6;
     // Egress filter configurations.
     Filter.Config egress_filters = 7;
+    // Minimum number of result to be returned.
+    uint32 minimum_num = 8 [ (validate.rules).uint32.gte = 0 ];
```

Comment:
```suggestion
    uint32 min_num = 8 [ (validate.rules).uint32.gte = 0 ];
```
How about it? It's just idea,

Reply:
yes!  your suggestion is more simple! it looks good to me!

Reply:
I have fixed: https://github.com/vdaas/vald/pull/1546/commits/a62f56c2b564d3875c1c5818a5f38b4ba58fd486